src/server/raft_client.rs

```diff
@@ -834,6 +834,7 @@ pub struct RaftClient<S, R, E> {
     builder: ConnectionBuilder<S, R>,
     engine: PhantomData<E>,
     last_hash: (u64, u64),
+    store_whitelist: Vec<u64>,
```

Comment:
This is not shared by threads. It's cloned to every threads. So setting it once may not update the copy of other threads.

Technically, it should  kill the connection in `ConnectionPool` and replace it with a blackhole. So other threads will refresh its connection list and find it's blocked.

It will be easier if `CachedQueue` support `pause`.

Reply:
It looks that simply pause `CacheQueue` may not be enough. We should make sure that connections to blocked stores not be created in the future.